app/models/cookie_student.rb

```diff
@@ -15,6 +15,10 @@ def welcome_banner_mark_as_viewed!
     }
   end
 
+  delegate :subjects, to: :degree, prefix: true
+
+  delegate :subject_groups, to: :degree, prefix: true
```

Comment:
nit: podemos ponerlos todos juntos

```suggestion
  delegate :subject_groups, :subjects, to: :degree, prefix: true
```